pints/toy/_stochastic_logistic_model.py

```diff
@@ -0,0 +1,132 @@
+#
+# Stochastic degradation toy model.
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import numpy as np
+from scipy.interpolate import interp1d
+import pints
+
+from . import ToyModel
+
+
+class StochasticLogisticModel(pints.ForwardModel, ToyModel):
+    r"""
```

Comment:
We need docstrings that provide full information (including references) for the model. At a minimum, this should include the model equations and description of how the model works. It should also describe the parameters that the model takes as inputs. See the deterministic logistic model or the Goodwin-oscillator for examples of this.

Reply:
@ben18785 Do you remember which paper was it? It says "Simpson et al. 2019" in the note, but wasn't easy to search...